packages/mds-audit/validators.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TelemetrySchema } from '@mds-core/mds-ingest-service'
+import { providers } from '@mds-core/mds-providers'
+import { SchemaValidator } from '@mds-core/mds-schema-validators'
+import { AUDIT_EVENT_TYPES, Telemetry, VEHICLE_EVENTS } from '@mds-core/mds-types'
+import {
+  AuditApiAuditEndRequest,
+  AuditApiAuditNoteRequest,
+  AuditApiAuditStartRequest,
+  AuditApiVehicleEventRequest
+} from './types'
+
+const uuidSchema = { type: 'string', format: 'uuid' }
+const timestampSchema = { type: 'integer', minimum: 100_000_000_000, maximum: 99_999_999_999_999 }
+
+export const { isValid: isAuditApiAuditStartRequestValid } = SchemaValidator<AuditApiAuditStartRequest['body']>(
+  {
+    $id: 'AuditApiAuditStartRequest',
+    type: 'object',
+    properties: {
+      timestamp: timestampSchema,
+      provider_id: { ...uuidSchema, enum: Object.keys(providers) },
```

Comment:
Is this really not required?

Reply:
thanks! that one was missing from the list. Also, removed `audit_event_type` from the `AuditApiAuditStartRequest` definition, since the request doesn't use it.